test/controllers/fhir/r4/api_controller_test.rb

```diff
@@ -62,7 +62,8 @@ def setup_system_applications
       force_password_change: false,
       api_enabled: true,
       role: 'public_health_enroller',
-      is_api_proxy: true
+      is_api_proxy: true,
+      notes: ''
```

Comment:
same thing here, probably not necessary to define a blank value but wouldn't be a bad idea to test this